packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -619,42 +619,10 @@ contract VotingV2 is
         commitAndEmitEncryptedVote(identifier, time, "", hash, encryptedVote);
     }
 
-    /**
-     * @notice Sets the delegate of a voter. This delegate can vote on behalf of the staker. The staker will still own
-     * all staked balances, receive rewards and be slashed based on the actions of the delegate. Intended use is using a
-     * low-security available wallet for voting while keeping access to staked amounts secure by a more secure wallet.
-     * @param delegate the address of the delegate.
-     */
-    function setDelegate(address delegate) external nonReentrant() {
-        voterStakes[msg.sender].delegate = delegate;
```

Comment:
I thought it made more sense for this function to be in `Staker` given that the state is in that last contract.